### DISCUSSION TITLE
Auto Subtitles

### DISCUSSION THREAD

**Discussion by AkoreJepade** [alexbelgium/hassio-addons#1512] [Q&A]
New to Jellyfin but I managed to get it up and running in Home Assistant and even managed to get Jellyfin app on my Samsung TV. Only thing I need some help with is Subtitles. I need Spanish subtitles on all my media (so that my wife doesn't divorce me) and I set it up in settings, but it has not downloaded them for my shows or movies. I did notice an Plugins section and perhaps that is what I am missing? 
![JellyfinSubs](https://github.com/user-attachments/assets/aa69d306-f040-4e8a-98fb-d2e6bdd2d1d2)
![JellyfinPlugins](https://github.com/user-attachments/assets/e6abdeaa-d200-48b1-8f9f-d8cb3251749c)

**Comment by alexbelgium** [comment]
Hi, sorry I don't use jellyfin but emby. However it should be fairly similar : that you need to install from within jellyin the subtitles addon (https://jellyfin.org/docs/general/server/plugins/open-subtitles/ ?), have an account on the website of the subtitles downloader, and set the addon. I think it should be from the "catalog" option just below the My Plugins

**Reply by AkoreJepade** [reply]
Thank you! I have it installed and logged in. Will see how it works.
